### PR TITLE
Auto Generate acceptance entry in travis matrix

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -32,6 +32,28 @@ matrix:
 <%     end -%>
 <%   end -%>
 <% end -%>
+<% if File.file?('modules/puppet-odoo/metadata.json') -%>
```
Comment (mcanevet) [inline moduleroot/.travis.yml]: @raphink any idea on how to lookup for the current module?

**Comment by mcanevet** [comment]
Will do this for example for `puppet-archive`:
```
--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,21 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 2.0.0
     env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
+  - env: BEAKER_set="debian-6-x86_64-openstack"
+    bundler_args:
+    script: '[ -z "${OS_AUTH_URL}" -o -z "${OS_USERNAME}" -o -z "${OS_PASSWORD}" -o -z "${OS_TENANT_NAME}" ] || bundle exec rspec spec/acceptance'
+  - env: BEAKER_set="debian-7-x86_64-openstack"
+    bundler_args:
+    script: '[ -z "${OS_AUTH_URL}" -o -z "${OS_USERNAME}" -o -z "${OS_PASSWORD}" -o -z "${OS_TENANT_NAME}" ] || bundle exec rspec spec/acceptance'
+  - env: BEAKER_set="redhat-5-x86_64-openstack"
+    bundler_args:
+    script: '[ -z "${OS_AUTH_URL}" -o -z "${OS_USERNAME}" -o -z "${OS_PASSWORD}" -o -z "${OS_TENANT_NAME}" ] || bundle exec rspec spec/acceptance'
+  - env: BEAKER_set="redhat-6-x86_64-openstack"
+    bundler_args:
+    script: '[ -z "${OS_AUTH_URL}" -o -z "${OS_USERNAME}" -o -z "${OS_PASSWORD}" -o -z "${OS_TENANT_NAME}" ] || bundle exec rspec spec/acceptance'
+  - env: BEAKER_set="redhat-7-x86_64-openstack"
+    bundler_args:
+    script: '[ -z "${OS_AUTH_URL}" -o -z "${OS_USERNAME}" -o -z "${OS_PASSWORD}" -o -z "${OS_TENANT_NAME}" ] || bundle exec rspec spec/acceptance'
 notifications:
   email: false
 deploy:
```

**Comment by raphink** [comment]
cool, in a scary way :+1:

**Comment by raphink** [comment]
@mcanevet Actually, this generates `redhat-*` entries, where we had `centos-*` before…

**Pull request merged**